editor/js/Viewport.ViewHelper.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * @author Mugen87 / https://github.com/Mugen87
+ */
+
+import * as THREE from '../../build/three.module.js';
+
+function ViewHelper() {
+
+	THREE.Object3D.call( this );
+
+	var camera = new THREE.OrthographicCamera( - 2, 2, 2, - 2, 0, 4 );
+	camera.position.set( 0, 0, 2 );
+
+	var axesHelper = new THREE.AxesHelper();
+	this.add( axesHelper );
+
+	var geometry = new THREE.SphereBufferGeometry( 0.2 );
+
+	var posXAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0xff0000, transparent: true } ) );
+	var posYAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0x00ff00, transparent: true } ) );
+	var posZAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0x0000ff, transparent: true } ) );
+	var negXAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0xff0000, transparent: true } ) );
+	var negYAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0x00ff00, transparent: true } ) );
+	var negZAxisHelper = new THREE.Mesh( geometry, new THREE.MeshBasicMaterial( { color: 0x0000ff, transparent: true } ) );
```

Comment:
It could make sense to replace the meshes with sprites (using generated textures) later on. In this way, it's possible to put X, Y and Z letters on the cardinal axes similar to Blender. Since sprites support raycasting, it should be also possible to implement interaction.